test/tests/functional/pbs_snapshot_unittest.py

```diff
@@ -933,12 +933,11 @@ def test_basic_option(self):
                     if not fpath.startswith(sched_priv_dir):
                         self.fail("Unexpected file " + fpath + " captured")
 
-    @classmethod
-    def tearDownClass(self):
+    def tearDown(self):
```

Comment:
Why did you change this?

Reply:
Yes! Can you tell me how to get rid of that? I just assumed that we can no longer override tearDownClass() in PTL and was going to file a ticket for it.

Reply:
There is a ticket for it already. But I think it's okay if you keep these changes. As you explained, it will delete snapshots after every test case as opposed to at the end of the test suite, which is fine.